File: src/main/java/me/lutto/questuhc/commands/tabcompleters/ArenaTabCompleter.java
package me.lutto.questuhc.commands.tabcompleters;

import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.command.TabCompleter;
import org.bukkit.entity.Player;
import org.bukkit.util.StringUtil;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class ArenaTabCompleter implements TabCompleter {

    @Override
    public List<String> onTabComplete(CommandSender sender, Command command, String label, String[] args) {

        if (args.length == 1) {
            return StringUtil.copyPartialMatches(args[0], Arrays.asList("list", "join", "leave"), new ArrayList<>());
        } else if (args.length == 2) {
            return StringUtil.copyPartialMatches(args[1], Arrays.asList("Type arena id here..."), new ArrayList<>());
        }

        return new ArrayList<>();

    }

}

File: src/main/kotlin/me/lutto/treasurebattle/commands/tabcompleters/ArenaTabCompleter.kt
package me.lutto.treasurebattle.commands.tabcompleters

import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.command.TabCompleter
import org.bukkit.util.StringUtil

class ArenaTabCompleter : TabCompleter {
    override fun onTabComplete(sender: CommandSender, command: Command, label: String, args: Array<String>): List<String>? {

        if (args.size == 1) {
            return StringUtil.copyPartialMatches(args[0], mutableListOf("list", "join", "leave"), ArrayList())
        } else if (args.size == 2) {
            return StringUtil.copyPartialMatches(args[1], mutableListOf("Type arena id here..."), ArrayList())
        }

        return ArrayList()

    }
}
